docker-pulumi/solutions/hello-dbc-done/index.ts

```diff
@@ -14,7 +14,7 @@
 //   !! You *must* complete the DBC builder setup steps @ https://docs.docker.com/build/cloud/setup/#steps
 // - Docker Desktop / CLI
 // - Pulumi CLI (https://www.pulumi.com/docs/get-started/install/)
-// - *Recommended* Pulumi Cloud account (https://app.pulumi.com/signup)
+// - *Recommended* Pulumi Cloud account (https://app.pulumi.com/signup?utm_source=GitHub&utm_medium=referral&utm_campaign=workshops)
```

Comment:
Per previous comment, I think we should let the docs be the docs and let the code be the code and refrain from links in comments. (Not relevant to this PR - I know you're just adding the codes where they are needed.)

Reply:
I'll make this change.